src/openfermion/ops/_symbolic_operator_test.py

```diff
@@ -151,6 +151,17 @@ def test_init_str(self):
         self.assertIn(correct, fermion_op.terms)
         self.assertEqual(fermion_op.terms[correct], -1.0)
 
+    def test_init_long_str(self):
+        fermion_op = DummyOperator1(
+                '-2.0 [0^ 1] +\n\n -1.0[ 2^ 3 ] - []', -1.)
+        correct = \
```

Comment:
I could be wrong, but I believe the way you are escaping to a new line here is a pep8 violation. Can you please run the pep8 linter to check?

Reply:
The pep8 linter says it's ok.